src/NLog/Targets/JsonSerializeOptions.cs

```diff
@@ -89,5 +89,15 @@ public JsonSerializeOptions()
             QuoteKeys = true;
             MaxRecursionLimit = 10;
         }
+
+        /// <summary>Registers a type<see cref="T:System.Type" /> to be excluded from serialization.</summary>
```

Comment:
I don't believe these methods belong in this class, but I couldn't find a better class. Any suggestions?